### ISSUE TITLE
Login via ssh, using key-file

### ISSUE THREAD

**Issue by BigLes** [TeamSQL/desktop-app#97]
OS: Linux x64 4.4.0-75-generic
VERSION: 0.9.91
---------------
Database Type: No applicable
Try to connect to ssh, using private key-file (id_rsa). Private key requests password to be provided, but there is no field for providing password to the key (only for the user). I tried to put pass of my id_rsa file to field named "Password", but it didn't work (I predict, that this is field for user password only, not the key-file).
Also, the error, when I try to connect to DB:
```
Uncaught Exception:
Error: Encrypted private key detected, but no passphrase given
    at Client.connect
```

**Comment by sanyer** [comment]
Wow, it's August and still no answer. Sweet.

**Comment by abacigil** [comment]
Ready for the next release

**Comment by abacigil** [comment]
Fixed on v1.4.122
